src/themes/whatsapp.scss

```
@@ -0,0 +1,192 @@
+  --gooey-whatsapp-text: #0a0a0a;
+  --gooey-whatsapp-muted: rgb(0 0 0 / 50%);
+  --gooey-message-content-max-width: 1000px;
+  --gooey-message-content-padding: 16px;
+  --gooey-outgoing-message-max-width: min(700px, calc(100% - 32px));
+  --gooey-button-outlined-background: #ffffff;
+  --gooey-button-outlined-border: transparent;
+  --gooey-button-hover: #f0f2f5;
+  --gooey-button-active: #e4e7ea;
+  --gooey-button-focus: var(--gooey-whatsapp-muted);
+  --gooey-tool-call-background: #ffffff;
+  // Declared once: each literal url() here is inlined as its own ~540KB data
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)